manifests/install.pp

```diff
@@ -4,8 +4,10 @@
 class zram_generator::install (
   Enum['installed', 'absent'] $install_defaults = $zram_generator::install_defaults,
 ) {
-  package { 'zram-generator-defaults':
-    ensure => $install_defaults,
+  unless $facts['os']['name'] == 'Archlinux' {
```

Comment:
I'm not a fan of OS specific conditionals in code. Would it make sense to have a `manage_defaults` parameter instead?

Reply:
updated it

Reply:
@ekohl thinking about it, by default ensure is set to absent for zram-generator-defaults. We could keep the code like it is on master and it works out of the box, but will fail on Arch Linux when someone sets it to installed. should we catch that and call fail()?